app/components/upcoming-project-card.js

```diff
@@ -10,6 +11,14 @@ export default class UpcomingProjectCardComponent extends Component {
 
   showPopup = false;
 
+  @computed('project.publicReviewPlannedStartDate')
```

Comment:
@hannahkates should be `@computed('assignment.publicReviewPlannedStartDate')` here